### ISSUE TITLE
Dropbox icon not visible with Kde Dark Breeze theme

### ISSUE THREAD

**Issue by lucazade** [keeferrourke/la-capitaine-icon-theme#261]
Dropbox icon is not visible when using Kde Dark Breeze theme
![screenshot_20171101_100654](https://user-images.githubusercontent.com/1557010/32267763-a38b6ff4-beec-11e7-9c5c-e6a5b3f9aa82.png)

Using a Light theme the icon works as expected

**Comment by keeferrourke** [comment]
Update pushed. Make sure you have run the configuration script and selected to the dark panel variant.

**Comment by lucazade** [comment]
tried latest commit and the configure script for dark panel but still no icon :(

**Comment by keeferrourke** [comment]
@lucazade did you try restarting dropbox after you updated the theme and reran configure? I know that NextCloud at the very least caches its icon and needs to be restarted to apply the icon change.

**Comment by lucazade** [comment]
Yes, I restarted dropbox after the update and reran configure script but no luck.

**Comment by keeferrourke** [comment]
Hmm... well Dropbox's state icon doesn't even appear to be themed by La Capitaine? Unless I just can't find the affected icon... Would you be able to provide the name of the indicator icon used by the Dropbox application?
